README.md

```diff
@@ -43,10 +45,13 @@ If your boost library headers are not in your PATH you can specify their locatio
 
 The ARCS+LINKS pipeline requires two input files:
 * Draft assembly fasta file
-* Interleaved linked reads file (Barcode sequence expected in the BX tag of the read header or in the form "@readname_barcode" ; Run [Long Ranger basic](https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger) on raw chromium reads to produce this interleaved file)
+* Reads file in fasta format `*.fa.gz` (or fastq format `*.fq.gz` if using long reads)
```

Comment:
I think you mean the opposite? Since short reads are normally in `.fq.gz` format?